src/components/app.js

```diff
@@ -24,12 +27,20 @@ class App extends Component {
     });
   };
 
+  handleSearch = (e) => {
+    this.setState({search: e.target.value});
+  };
+
   render() {
-    const { contactList } = this.state;
+    const { contactList, search } = this.state;
+
+    const filteredList = contactList.filter(contact => contact.name.includes(search));
```

Comment:
Can you implement filtering as util function with is receive contacts and search value as an arguments and returns filtered array?

Reply:
moved to separate method and call it inside.